include/nbl/asset/IDescriptorSetLayout.h

```diff
@@ -340,14 +342,14 @@ class IDescriptorSetLayout : public virtual core::IReferenceCounted  // TODO: tr
 
 	inline const CBindingRedirect& getDescriptorRedirect(const IDescriptor::E_TYPE type) const { return m_descriptorRedirects[static_cast<uint32_t>(type)]; }
 	inline const CBindingRedirect& getImmutableSamplerRedirect() const { return m_immutableSamplerRedirect; }
-	inline const CBindingRedirect& getMutableSamplerRedirect() const { return m_mutableSamplerRedirect; }
+	inline const CBindingRedirect& getMutableCombinedSamplerRedirect() const { return m_mutableCombinedSamplerRedirect; }
 
 	inline core::SRange<const core::smart_refctd_ptr<sampler_type>> getImmutableSamplers() const
```

Comment:
for the `getImmutable...` don't you need to distinguish between immutables from COMBINED and SAMPLER bindings?

Reply:
Not sure, I don't see why I would need to. I need a refresher on most of this but I don't recall needing to